host-interaction/registry/query/query-registry-entry.yml

```diff
@@ -1,19 +0,0 @@
-rule:
-  meta:
-    name: query registry entry
-    namespace: host-interaction/registry/query
-    author: william.ballenthin@fireeye.com
-    scope: function
-    att&ck:
-      - Discovery::Query Registry [T1012]
-    examples:
-      - BFB9B5391A13D0AFD787E87AB90F14F5:0x13145B5A
-      - Practical Malware Analysis Lab 03-02.dll_:0x100047AD
-  features:
-    - and:
-      - or:
-        - match: create registry key
-        - match: open registry key
-      - or:
-        - api: advapi32.RegQueryValue
-        - api: advapi32.RegQueryValueEx
```

Comment:
appeared to be iteration of `query-registry-value.yml`; removed to eliminate duplication